src/lib.rs

```diff
@@ -145,6 +148,25 @@ pub struct Node {
     pub id: String,
     /// List of opaque identifiers for this node's dependencies
     pub dependencies: Vec<String>,
+
+    /// Dependencies in a structured format
+    #[serde(default)]
+    pub deps: Vec<NodeDep>,
+    /// Features provided by the crate
```

Comment:
I think this would be better to say that it is the set of features *enabled* on the crate.  The set of features available is something different.